### PR TITLE
Updates Metamask Page with new steps

### PR DESCRIPTION
**Pull request by albertov19**
### Description

Adds new Metamask flow and revamps the page a bit

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images

#### Items to be Updated

Delete all of the metamask/metamask-* images and replace with the ones in the metamask/new/ directory

### CODE REVIEW

File: tokens/connect/metamask.md [hunk 1]
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
...
+3. **Chain ID** - Chain ID
+4. **Symbol** - (optional) symbol of the native token of the network. For example, for Moonbeam, the value would be **GLMR**
+5. **Block Explorer** - (optional) URL of the [block explorer](https://docs.moonbeam.network/builders/get-started/explorers/){target=_blank}
+6. Once you've verified all the information, click on **Save**
 
-You can also try sending some tokens with MetaMask. You will need two accounts for this example, so if you need to create another one you can do so now. Once you have two accounts, click **Send** to initiate the transfer. Select the **Transfer between my accounts** option, transfer 100 tokens, and leave all other settings as they are:
+![Add network in Metamask](/images/tokens/connect/metamask/new/metamask-8.png)
 
-![Initiating a token transfer](/images/tokens/connect/metamask/metamask-8.png)
+Once you've added the network, you'll be redirected to a screen stating that you've successfully added a network. Furthermore, you'll be prompted a button to **Switch to Moonbase Alpha**, the network added in this example.
```
Comment (eshaben) [inline tokens/connect/metamask.md:139]:
```suggestion
Once you've added the network, you'll be redirected to a screen stating that you've successfully added a network. Furthermore, you'll be prompted to **Switch to Moonbase Alpha**, the network added in this example.
```

File: tokens/connect/metamask.md [hunk 2]
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
...
 
-Once you have submitted the transaction, you will see it **pending** until it is confirmed, as shown in the following image:
+![Successfully added a network in Metamask](/images/tokens/connect/metamask/new/metamask-9.png)
 
-![Transaction confirmation](/images/tokens/connect/metamask/metamask-9.png)
+## Interact with the Network {: #interact-with-network } 
 
-Note that the Account 2 balance has been decreased by the sent amount + gas fees. Flipping over to Account 1, you can see the 100 sent tokens have arrived:
+Once you've [connected Metamask](#connect-metamask-to-moonbeam) to any Moonbeam-based Network, you can start using your wallet by:
```
Comment (eshaben) [inline tokens/connect/metamask.md:145]:
```suggestion
Once you've [connected Metamask](#connect-metamask-to-moonbeam) to any Moonbeam-based network, you can start using your wallet by:
```

File: tokens/connect/metamask.md [hunk 3]
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
...
+### Initiate a Transfer { #initiate-a-transfer } 
 
-![Moonbeam Development Node](/images/tokens/connect/metamask/metamask-11.png)
+This section showcases how to do a simple token transfer to another address as an example of using Metamask with Moonbeam.
+
+To do so, take the following steps:
+
+1. Ensure you are connected to the correct network
+2. Ensure you have selected the account you want to use for the transfer
+3. In the main screen of your Metamask wallet, click on **Send**
```
Comment (eshaben) [inline tokens/connect/metamask.md:159]:
```suggestion
3. On the main screen of your Metamask wallet, click on **Send**
```

File: tokens/connect/metamask.md [hunk 4]
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
...
+
+![Initiate balance transfer in Metamask](/images/tokens/connect/metamask/new/metamask-10.png)
+
+Next, you can enter the address to which you want to send the tokens. For this example, a wallet that has already been imported to Metamask is selected, known as **Bob**.
+
+![Select account to send tokens to in Metamask](/images/tokens/connect/metamask/new/metamask-11.png)
+
+In the next screen, take the following steps:
```
Comment (eshaben) [inline tokens/connect/metamask.md:167]:
```suggestion
On the next screen, take the following steps:
```

**Review by eshaben — COMMENTED** [review]
Thanks for updating this page! It needed some love!

Mostly tiny things, but I did notice that some of the images (some are worse than others) have white around the edges, which kind of stands out on the dark background. There are also a good number of images with a grey line that goes like half way across the bottom of the image.

I could probably update these tomorrow (Friday), if you don't get to it first

**Comment by albertov19** [comment]
@eshaben thanks for the thorough review.

The weird lines in the images is due to the screenshot process in my laptop, got to look into that because it is very annoying haha. 

I've addressed all the feedback

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**